tasks/publish-pyxis-repository/tests/mocks.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+set -eux
+
+# mocks to be injected into task step scripts
+
+function curl() {
+  echo Mock cosign called with: $* >&2
+  echo $* >> $(workspaces.data.path)/mock_curl.txt
+
+  CALL_ID=$(cat $(workspaces.data.path)/mock_curl.txt | wc -l)
+
+  if [[ "$*" == "--retry 5 --key /tmp/key --cert /tmp/crt https://pyxis.api.redhat.com/v1/repositories/registry/${PYXIS_REGISTRY}/repository/my-product/my-image"?" -X GET" ]]
+  then
+    # condition for missing Pyxis Repository scenario
+    if [[ "$*" == *"my-image9"* ]]; then
+      echo '{"detail": "Document in containerRepository not found.", "status": 404, "title": "Not Found", "type": "about:blank", "trace_id": "0x4d7be17d142d24c5f2b10b5f1745cc89"}'
+    else
+      echo '{"_id": "'$CALL_ID'"}'
+    fi
+  elif [[ "$*" == '--retry 5 --key /tmp/key --cert /tmp/crt https://pyxis.api.redhat.com/v1/repositories/id/'?' -X PATCH -H Content-Type: application/json --data-binary {"published": true}' ]]
+  then
+    :
```

Comment:
what does this do?

Reply:
Nothing. It's a no-op. Like `pass` in python.

I just tried and it seems it works even without it - I assumed you do need a command. I guess I still prefer to have an explicit no-op command. But I can remove it if you'd prefer that.

Reply:
Or a comment could be nice - I thought it was a typo at first

Reply:
OK, added a comment.